test/api/v3/integration/groups/GET-groups.test.js

```diff
@@ -0,0 +1,97 @@
+import {
+  generateUser,
+  resetHabiticaDB,
+  generateGroup,
+} from '../../../../helpers/api-v3-integration.helper';
+
+describe('GET /groups', () => {
+  let user;
+  const NUMBER_OF_PUBLIC_GUILDS = 3;
+  const NUMBER_OF_USERS_PRIVATE_GUILDS = 1;
+  const NUMBER_OF_GUILDS_USER_CAN_VIEW = 6;
```

Comment:
@TheHollidayInn I'm not sure why user has access to 6 groups: party, tavern, 2 public guilds and a private guild. It shouldn't be 5?